packages/console/src/pages/Connectors/components/CreateForm/index.module.scss

```diff
@@ -33,6 +60,8 @@
 
         .name {
           font: var(--font-subhead-2);
+          @include _.multi-line-ellipsis(1);
+          padding-right: _.unit(12); /* For check mark and added label */
```

Comment:
I think we can make the `Added` tag an inline `<span />` along with the connector name. So that it won't end up with something like this:

<img width="215" alt="image" src="https://user-images.githubusercontent.com/12833674/196171495-c92f0f88-6e4e-438b-a8c6-b7130ec9310f.png">

We can also add `padding-right` only when it is selected. Anyways, there's still room for improvement.

Reply:
Good suggestion! I chose the second and updated